### ISSUE TITLE
How to predict unlabeling test data?

### ISSUE THREAD

**Issue by qute012** [noahchalifour/baidu-deepspeech2#4]
Hi. I'm Studying Speech Recognition. I have some question.
This model takes 4 inputs,  below code.
```python
self.model = tf.keras.Model(inputs=[input_data, labels, input_length, label_length], outputs=[loss_out])
```
If this model predicts test case that is unlabeled, i can't give 'labels', 'label_length' as inputs.
How can i do?
